### PR TITLE
Add MAUI and WPF Blazor Hybrid XAML namespaces

### LINKED ISSUE #4303: Define a default namespace for .NET MAUI Blazor controls similar to that of Community Toolkit
### Description

Request to consider defining a default HTTP alias for .NET MAUI Blazor controls similar to that of Community Toolkit so that it becomes easier to manage them while using in XAML code

### Public API Changes

Instead of the complex xmlns definition with clr-namespace and assembly names as shown in the below example, when defined with appropriate HTTP alias this will become easy to manage.

such as `xmlns:b="http://..."`

```xaml
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:b="clr-namespace:Microsoft.AspNetCore.Components.WebView.Maui;assembly=Microsoft.AspNetCore.Components.WebView.Maui"
             xmlns:local="clr-namespace:MauiBlazorApp"
             x:Class="MauiBlazorApp.MainPage"
             BackgroundColor="{DynamicResource PageBackgroundColor}">
    <b:BlazorWebView HostPage="wwwroot/index.html">
        <b:BlazorWebView.RootComponents>
            <b:RootComponent Selector="#app" ComponentType="{x:Type local:Main}" />
        </b:BlazorWebView.RootComponents>
    </b:BlazorWebView>
</ContentPage>
```

### Intended Use-Case

Allows for quick definition and better source management.

This will result in quite like the community toolkit definition:

xct="http://xamarin.com/schemas/2020/toolkit"

### PR DESCRIPTION
**Pull request by MackinnonBuck**
### Description of Change

Added the following XAML namespaces for Blazor Hybrid:
MAUI: `http://schemas.microsoft.com/dotnet/2021/maui` (same as other MAUI controls)
WPF: `http://schemas.microsoft.com/winfx/2006/xaml/presentation/blazor`

The namespace names are subject to further discussion in this PR.

The templates and sample projects have been updated to reflect this change.

### Issues Fixed

Fixes #4303

### CODE REVIEW

File: src/Controls/samples/Controls.Sample/Pages/Blazor/BlazorPage.xaml [hunk 1]
```diff
@@ -7,8 +7,6 @@
     Title="Blazor">
 
     <!--
```
Comment (SteveSandersonMS) [inline src/Controls/samples/Controls.Sample/Pages/Blazor/BlazorPage.xaml:9]: Any idea why this is all commented out?

**Review by SteveSandersonMS — APPROVED** [review]
Much nicer. Great!

**Review by Eilon — APPROVED** [review]
I approve as-is, but per @SteveSandersonMS 's comment, it's probably worth using XAML in the Controls Sample here now.

**Review by TanayParikh — APPROVED** [review]
🎉

File: src/BlazorWebView/src/Maui/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Internals;
+
+[assembly: Preserve]
+
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/2021/maui", "Microsoft.AspNetCore.Components.WebView.Maui")]
```
Comment (StephaneDelcroix) [inline src/BlazorWebView/src/Maui/Properties/AssemblyInfo.cs:6]: are we 100% sure there's no duplicate types in that namespace ? cause it'd cause tooling issues

Comment (StephaneDelcroix) [reply src/BlazorWebView/src/Maui/Properties/AssemblyInfo.cs:6]: if there's none, go ahead and merge

Comment (Eilon) [reply src/BlazorWebView/src/Maui/Properties/AssemblyInfo.cs:6]: Nope, just 1 control plus a few related types. No one else uses that namespace.

**Pull request merged**